developers_chamber/slack_utils.py

```diff
@@ -0,0 +1,35 @@
+import subprocess
+
+from slack_sdk import WebClient
+
+
+def run_sh(cmd: str) -> list:
+    return [
+        line for line in subprocess.run(
+            cmd, shell=True, check=True, stdout=subprocess.PIPE
+        ).stdout.decode('utf-8').rstrip().split('\n')
+    ]
+
+
+def upload_new_migration(token, channel, diff_branch):
+    client = WebClient(token=token)
+    current_branch = run_sh('git branch --show-current')[0]
+
+    migration_files = run_sh(
+        f'git diff --color --name-only --diff-filter=A {diff_branch} {current_branch}'
```

Comment:
This is security issue, `diff_branch` is user input, i.e. arbitrary code could be executed here.

Reply:
Fixed.